core/templates/dev/head/pages/exploration_player/conversation_skin_directive.html

```diff
@@ -60,12 +69,14 @@
          class="conversation-skin-supplemental-card-container">
       <supplemental-card on-click-continue-button="showUpcomingCard()"
                          is-learn-again-button="isLearnAgainButton()"
-                         on-submit-answer="submitAnswer(answer, rulesService)">
+                         on-submit-answer="submitAnswer(answer, rulesService)"
+                         ng-if="!conceptCardShown">
       </supplemental-card>
       <div class="conversation-skin-supplemental-nav" ng-if="isSupplementalNavShown()">
         <md-button type="submit" class="oppia-learner-confirm-button protractor-test-submit-answer-button conversation-skin-supplemental-nav-submit-button"
                    ng-click="submitAnswerFromProgressNav()"
                    ng-disabled="!interactionAnswerIsValid"
+                   ng-if="!conceptCardShown"
```

Comment:
Perhaps this should be part of the ng-if in the parent.

Reply:
Done.